c++/src/kj/mutex.h

```diff
@@ -197,6 +197,8 @@ class Mutex {
 #elif _WIN32 || __CYGWIN__
   uintptr_t srwLock;  // Actually an SRWLOCK, but don't want to #include <windows.h> in header.
 
+#elif defined(KJ_NO_THREADS)
+  uint32_t dummy_mutex;
```

Comment:
Please keep the naming style consistent -- this should be `dummyMutex`.

What does this variable do, though? It looks like it is unused. Can you just delete it?

Similarly for the other "dummy" fields below.

Reply:
There is a `static_assert` that explodes if the Mutex has a too small size.  I'll paste it in.